assets/app/scripts/services/auth.js

```diff
@@ -74,6 +74,12 @@ angular.module('openshiftConsole')
     var logoutService = loadService($injector, _logoutService, "AuthServiceProvider.LogoutService()");
 
     return {
+      
+      // Returns the configured user store
+      UserStore: function() {
+        return _userStore;
```

Comment:
if we're exposing this, just return `userStore` so the caller doesn't have to use `$injector.get()`